plugin/ims_lti/src/Service/Resource/LtiResultsResource.php

```diff
@@ -0,0 +1,235 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+use Chamilo\CoreBundle\Entity\GradebookResult;
+use Chamilo\PluginBundle\Entity\ImsLti\LineItem;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Doctrine\ORM\TransactionRequiredException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\Exception\UnsupportedMediaTypeHttpException;
+
+/**
+ * Class LtiResultsResource.
+ */
+class LtiResultsResource extends LtiAdvantageServiceResource
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class LtiResultsResource has a coupling between objects value of 14. Consider to reduce the number of dependencies under 13.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=4799654)